client/components/forms/form-country-select/index.jsx

```diff
@@ -34,7 +34,7 @@ export default localize( React.createClass( {
 
 		return (
 			<select
-				{ ...omit( this.props, 'className' ) }
+				{ ...omit( this.props, [ 'className', 'countriesList', 'translate', 'moment', 'numberFormat' ] ) }
```

Comment:
Yup, I much prefer being explicit like that - otherwise, if a new prop will get added to the parent component (but not the child component), this problem could resurface again.